### PR TITLE
Part 2 of Item #79 - only affecting section 040

### PR DESCRIPTION
**Pull request by laurie-tyz**
Part 2 of Item #79  - only affecting section 040
The focus is on remediation, patches and fixes
Some changes from Patches to remediation
some changes from fixes to remediation
Rolled back all changes from fixes to patches.

Will create a separate branch to further discuss fixes/patches/remediation with @cgyarbrough .

Previously reviewed by Jono and @cgyarbrough

### CODE REVIEW

File: doc/version_1/040_treesForVulMgmt.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Table 3: Proposed Meaning for Deployer Priority Outcomes
 | ---------------- | -------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Defer            | Do not act at present.                                                                                                                          |
 | Scheduled        | Act during regularly scheduled maintenance time.                                                                                                   |
-| Out-of-cycle     | Act more quickly than usual to apply the fix out-of-cycle, during the next available opportunity, working overtime if necessary.                    |
+| Out-of-cycle     | Act more quickly than usual to apply the remediation out-of-cycle, during the next available opportunity, working overtime if necessary.                    |
```
Comment (j---) [inline doc/version_1/040_treesForVulMgmt.md:59]: Suggest s/apply the remediation/act/ here. I don't think this definition should prefer remediation over mitigation as the action the deployer might take.

File: doc/version_1/040_treesForVulMgmt.md [hunk 2]
```diff
@@ -30,21 +30,21 @@ To further clairify terms, "Remediaton occurs when the vulnerability is eliminat
...
 
 Table 2: Proposed Meaning for Supplier Priority Outcomes
 
 | Supplier Priority | Description |
 | ------------------ | ----------- |
 | Defer              | Do not work on the patch at present.   |
 | Scheduled          | Develop a fix within regularly scheduled maintenance using supplier resources as normal.  |
-| Out-of-Cycle       | Develop a fix out-of-cycle, taking resources away from other projects and releasing the fix as a security patch when it is ready. |
+| Out-of-Cycle       | Develop remediation out-of-cycle, taking resources away from other projects and releasing the fix as a security patch when it is ready. |
```
Comment (j---) [inline doc/version_1/040_treesForVulMgmt.md:41]: Probably "mitigation or remediation" rather than "remediation"

**Review by j--- — APPROVED** [review]
This looks good, though there are two minor comments that I think should be addressed before merging.

File: doc/version_1/040_treesForVulMgmt.md (hunk 2 above)
Comment (laurie-tyz) [reply doc/version_1/040_treesForVulMgmt.md]: completed

**Pull request merged**